File: gestion_personnel/src/main/java/com/cdaprojet/gestion_personnel/model/SignupRequest.java
package com.cdaprojet.gestion_personnel.model;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@AllArgsConstructor
@NoArgsConstructor
@Data
public class SignupRequest {
    
    private String secondname;
    private String firstname;
    private String email;
    private String password;
    private String roleName;

}

File: gestion_personnel_front/src/app/models/signup-request/signup-request.ts
export class SignupRequest {

    secondname: string;
    firstname: string;
    email: string;
    pwd: string;
    roleName: string;

    constructor(
        secondname: string, 
        firstname: string, 
        email: string, 
        pwd: string, 
        roleName: string
    ) {
        this.secondname = secondname;
        this.firstname = firstname;
        this.email = email;
        this.pwd = pwd;
        this.roleName = roleName;
    }
}